Automattic/wp-calypso
# Jetpack connect plans page

This PR creates a new plans page to redirect the users after they connect their jetpack site via jetpack-connect. 

![image](https://cloud.githubusercontent.com/assets/1554855/15047880/32f08db8-12e7-11e6-87b6-7ffc4b0c7099.png)

![untitled screencast 12](https://cloud.githubusercontent.com/assets/1554855/15047887/3aed1e8c-12e7-11e6-870d-ff82683cc443.gif)

Also, this fix https://github.com/Automattic/wp-calypso/issues/5149 (as a side effect)

How to test
========
1. You need a .org site running the latest master version of jetpack
2. Disconnect your Jetpack from .com, if needed
3. Go to http://calypso.localhost:3000/jetpack/connect and enter your site
4. After a while, you should be taken to that new plans page

ping @roccotripaldi @rickybanister and someone who has worked with plans before... @drewblaisdell ? @stephanethomas ?
